test/e2e/network/dns_configmap.go

```diff
@@ -229,10 +219,20 @@ func (t *dnsPtrFwdTest) run() {
 }
 
 var _ = SIGDescribe("DNS configMap nameserver", func() {
-	t := &dnsPtrFwdTest{dnsTestCommon: newDnsTestCommon()}
-	BeforeEach(func() { t.c = t.f.ClientSet })
+
+	nsTest := &dnsNameserverTest{dnsTestCommon: newDnsTestCommon()}
+	fwdTest := &dnsPtrFwdTest{dnsTestCommon: newDnsTestCommon()}
+
+	BeforeEach(func() {
+		nsTest.c = nsTest.f.ClientSet
+		fwdTest.c = fwdTest.f.ClientSet
```

Comment:
Actually, seems unnecessary to initialize both tests in BeforeEach, what about initialize them in each It block instead?

Reply:
Done.